files/default/handler/audit_report.rb

```diff
@@ -41,7 +41,8 @@ def report
         end
         quiet = node['audit']['quiet']
         fetcher = node['audit']['fetcher']
-        attributes = node['audit']['attributes'].to_h
+
+        attributes = node.run_state['audit_attributes'].to_h
```

Comment:
Going from a Mash to a Hash going to cause any issues? (I know it was a `.to_h` before, but I'm curious)

Reply:
Nobody reported issues with it and tests are showing the expected values.